pulp_docker/app/viewsets.py

```diff
@@ -24,6 +26,91 @@
 from . import models, serializers, tasks
 
 
+class ManifestListTagViewSet(ContentViewSet):
+    """
+    ViewSet for ManifestListTag.
+    """
+
+    endpoint_name = 'docker/manifest-list-tags'
+    queryset = models.ManifestListTag.objects.all()
+    serializer_class = serializers.ManifestListTagSerializer
+
+    @transaction.atomic
+    def create(self, request):
+        """
+        Create a new ManifestListTag from a request.
+        """
+        raise NotImplementedError()
```

Comment:
Since the parent ViewSet includes POST (create), it is necessary to override this-- and because the upload functionality is sufficiently complex, I think it should be done in a separate issue/PR. I'll file that issue before I merge.